quadrat/inc/customization.php

```diff
@@ -0,0 +1,102 @@
+<?php
+class CustomizerBridge {
+
+	private $theme_customizations;
+
+	function __construct() {
+		add_action( 'customize_register', array( $this, 'customizer_bridge_register' ) );
+		add_action( 'wp_enqueue_scripts', array( $this, 'customizer_bridge_output_variables' ) );
+		$this->theme_customizations = $this->get_theme_customizations();
+	}
+
+	function get_theme_customizations() {
+		$string  = file_get_contents( get_stylesheet_directory() . '/experimental-theme.json' );
+		$decoded = json_decode( $string );
+		return $decoded->settings->custom_meta;
+	}
+
+	/**
+	 * Enqueue color variables for customizer & frontend.
+	 */
+	function customizer_bridge_output_variables() {
+		wp_add_inline_style( 'customizer-style', $this->customizer_bridge_generate_custom_color_variables() );
+	}
+
+	/**
+	 * Render the customized variables.
+	 */
+	function customizer_bridge_generate_custom_color_variables( $context = null ) {
+
+		$css_variables = 'body {';
```

Comment:
targetting just the body is not enough, this doesn't work on the post editor

Reply:
Hrm...  We may need to target the body AND root:...

Originally this targeted only `root:` and that worked ok.  But between the original experiment and today the majority of Global Styles (on the view) are applied to the BODY not ROOT.  Thus applying these variables to ROOT they are simply overwritten by the global styles applied to the body (no matter the load order).

Reply:
yeah, old themes target both for the color annotations to work (Seedlet as an example) but I tried and it didn't work, even when I enqueued the css to show on the editor, I never saw the styles showing up at all.